nullaway/src/main/java/com/uber/nullaway/fixserialization/FixSerializationConfig.java

```diff
@@ -89,6 +89,17 @@ public FixSerializationConfig(
     serializer = new Serializer(this, initializeAdapter(SerializationAdapter.LATEST_VERSION));
   }
 
+  public DocumentBuilderFactory safeDocumentBuilderFactory() {
+    DocumentBuilderFactory dbf = DocumentBuilderFactory.newInstance();
+    dbf.setFeature("http://apache.org/xml/features/disallow-doctype-decl", true);
+    dbf.setFeature("http://apache.org/xml/features/nonvalidating/load-external-dtd", false);
+    dbf.setFeature("http://xml.org/sax/features/external-general-entities", false);
+    dbf.setFeature("http://xml.org/sax/features/external-parameter-entities", false);
+    dbf.setFeature("http://apache.org/xml/features/dom/create-entity-ref-nodes", false);
+    dbf.setFeature(XMLConstants.FEATURE_SECURE_PROCESSING, true);
+    return dbf;
+  }
+
```

Comment:
Once we make the `XMLUtil#safeDocumentBuilderFactory` method static, we can get rid of this method, and just call `XMLUtil.safeDocumentBuilderFactory()` below.